packages/api-explorer/src/index.jsx

```diff
@@ -40,6 +42,14 @@ function getDescription(oasFiles){
   return get(oasFiles, 'api-setting.info.description')
 }
 
+const panelStyle = {
+  margin: '5px 0px',
+  borderRadius: 5,
+  overflow: 'hidden',
+}
+
+const REDOC_CDN = 'https://cdn.jsdelivr.net/npm/redoc@2.0.0-rc.56/bundles/redoc.standalone.js'
```

Comment:
keeping out of this discussion whether it's ok to load it from a CDN, I'd not load a release candidate 🤔

Reply:
Since a couple of years, when v2 has been initially released, Redoc only has release candidate:
https://github.com/Redocly/redoc/releases